content/Main_Notebooks/2 - Pandas Data Structures.ipynb

```diff
@@ -2502,11 +2520,15 @@
     "\n",
     "However, right now we got only items in ID that were present in both dataframes.We can change this behavior with the keyword `how`. We have four simple options:\n",
     "- Inner: This is the default, where only items present in the matching column in both frames are included.\n",
-    "- Outer: All items are included.\n",
-    "- Left: All columns in the left dataframe (in the order you write them in `pd.merge()`) are kept, ones only present in the right dataframe are discarded.\n",
-    "- Right: The opposite of left.\n",
+    "- Outer: All rows from both dataframes are included.\n",
+    "- Left: All columns in the left dataframe (in the order you write them in `pd.merge()`) are kept. Ones only present in the right dataframe than don't match up with any rows in the left dataframe are discarded.\n",
```

Comment:
> Ones only present in the right dataframe than don't match up 3

I think that "than" was supposed to be a "that"

Reply:
yup, you're right